ghstack/config.py

```diff
@@ -130,9 +131,8 @@ def read_config(*, request_circle_token: bool = False) -> Config:  # noqa: C901
     else:
         default_project_dir = 'fbcode/caffe2'
 
-    if write_back:
-        config.write(open(os.path.expanduser('~/.ghstackrc'), 'w'))
-        logging.info("NB: configuration saved to ~/.ghstackrc")
+    config.write(open(config_path, 'w'))
```

Comment:
It's probably better not to write to the config file every time the you run ghstack; that increases the possibility of file corruption when really it shouldn't ever be possible.